Jenkinsfile

```diff
@@ -177,6 +177,7 @@ pipeline {
                   expression { return env.GITHUB_COMMENT?.contains('integration tests') }
                   expression { matchesPrLabel(label: 'ci:agent-integration') }
                   expression { return env.CHANGE_ID != null && !pullRequest.draft }
+                  not { changeRequest() }
```

Comment:
![image](https://user-images.githubusercontent.com/2871786/151214895-10278e6f-9181-45da-bc00-746e780afbfa.png)

Reply:
'Non-Application Server integration tests', 'Application Server integration tests', 'End-To-End Integration Tests' and `Bencharmks` are skipped in any branch (so far only `main`)

The idea with this PR is to keep the existing behaviour for PRs but enable those stages when merging into `main`.

Since I was not sure about the branch strategy, I played safe with `not { changeRequest() }` therefore there is no need to change the CI pipeline if a new branch is created for whatever reasons.

I'd prefer to use `not { changeRequest() }` , even though it's a negative condition

Reply:
I'm fine with keeping it as-is, even if it's negative (covid learned us that negative things aren't that bad after all :smile: ).